react/ProductList.js

```diff
@@ -10,7 +10,7 @@ import GapPaddingTypes, {
   getGapPaddingValues,
 } from './paddingEnum'
 import ShelfContent from './ShelfContent'
-import ShelfItem from './ShelfItem'
+import { ShelfItem } from './ShelfItem'
```

Comment:
This is wrong. Revert this.

Reply:
I exported without Pixel because other components needed to use ShelfItem.propTypes (exporting with Pixel override this field). But now I changed how they are getting these propTypes. :)